JDBCEmailRepository.java

```diff
@@ -0,0 +1,142 @@
+package com.example.demo.repository;
+
+import com.example.demo.Email;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+@Repository
+public class JDBCEmailRepository implements EmailRepository {
+
+    private Connection connection;
+    private Statement statement=null;
+    private ResultSet resultSet=null;
+    private ArrayList<Email> emails=new ArrayList<Email>();
+    private Email email=new Email();
+
+    @Autowired
+    public JDBCEmailRepository(Connection connection) {
+        this.connection = connection;
+    }
+
+
+    public ArrayList<Email> getActive()  {
+        emails.clear();
+
+        try {
+
+            statement = connection.createStatement();
+            String selectActiveQuery="Select * from email where isSent=false";
```

Comment:
Keep uppercase SQL code style.
It makes possible to mentally distinguish reserved words (commands) from identifiers.

```
SELECT * FROM email WHERE isSent = false
```